### PR TITLE
Proposed Changes for Contribution Guidelines - High Fidelity Development Fund

### PR DESCRIPTION
**Pull request by misslivirose**
Accounting for the new changes with the High Fidelity Development Fund to more easily facilitate external contributions.

cc @robin-k-wilson @RebeccaStankus @MiladNazeri @AndySeattle for additional feedback.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 Contributing
 ===
+Welcome to the world of content development in High Fidelity! This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).
+
+The High Fidelity Development Fund
+===
+The High Fidelity Development Fund is a way for members of our community can take on projects to extend the High Fidelity platform. There are dozens of features we’re planning to add, and we’re looking to developers in our community to sign up to create them. Of course, we’re planning to pay them for their hard work — in HFC, which they can convert to legal tender.
```
Comment (robin-k-wilson) [inline contributing.md]: Suggestions:
**Typo:** The High Fidelity Development Fund is a way for members of our community ~~can~~ **to** take on projects to extend the High Fidelity platform.

**Wordsmithing:** We're looking for developers in the community to sign up and create the dozens of new features we have planned. Of course, your hard work will be compensated - in HFC, which is convertible to legal tender.

File: contributing.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+
+We’ve set up a [public group on Telegram](https://t.me/highfidelityvr) which developers can use to contact us about these projects. We plan to regularly post new opportunities to the channel for people to select and bid for. The aim here is to reward our most dedicated members, to accelerate feature rollout, and to inject more activity into the economy, which in turn will benefit everyone.
```
Comment (robin-k-wilson) [inline contributing.md]: **Wordsmithing:** We ~~’ve set up~~ **have** a [public group on Telegram](https://t.me/highfidelityvr) which developers ~~can~~ use to contact us about these projects. We ~~plan to~~ regularly post new opportunities to the channel for people to select and bid for. The aim here is to reward our most dedicated members, to accelerate feature rollout, and to inject more activity into the economy, which in turn will benefit everyone.

File: contributing.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+
+In this first wave of the Fund, we’re committing 1,000,000 HFC to fund developers. Use the Telegram channel to bid on initiatives, propose new projects, and suggest new features for the platform. [Sign up here to learn more](https://t.me/highfidelityvr).
+
+Submitting Community Project Proposals
+===
+Please use the GitHub Issues tracking tab to submit a new issue or High Fidelity Developer Fund initiative. Tag these with the labels 'Community' and ensure that you are managing your proposals and updating the status accordingly as the projects are approved or worked on. We resolve the right to close issues that do not adhere to our guidelines or code of conduct, do not align with our community standards, or are left unresolved for extended time without update.
```
Comment (robin-k-wilson) [inline contributing.md]: 'Community' --> "Community"
resolve --> reserve

File: contributing.md [hunk 4]
```diff
@@ -33,6 +50,10 @@ Contributing
 
   *You can follow [GitHub's guide](https://help.github.com/articles/creating-a-pull-request) to find out how to create a pull request.*
   In your pull request, please include a brief test plan that can be run to validate the changes that you've made in your PR.
+  
+9. Tag your pull request with the appropriate labels for the project that you are working on so we can track progress with the High Fidelity Developer Fund projects and internal initiatives. PRs that are made by contributors outside of the High Fidelity staff team should be tagged 'Community'. If you would like specific feedback on a PR or an issue, please use the 'Feedback Welcome' tag.
```
Comment (robin-k-wilson) [inline contributing.md]: 'Community' --> "Community"
'Feedback Welcome' --> "Feedback Welcome"

File: contributing.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
 Contributing
 ===
+Welcome to the world of content development in High Fidelity! This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).
```
Comment (robin-k-wilson) [inline contributing.md]: Welcome to the world of content development in High Fidelity!

This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request **for compensated work** made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).

OR 

either as part of your own work or **through the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc) where developers are compensated for their work.**

Added: Spacing and "for compensated work"
I think highlighting compensation is desireable. If that is not our intention.

File: contributing.md [hunk 6]
```diff
@@ -1,5 +1,22 @@
 Contributing
```
Comment (robin-k-wilson) [inline contributing.md]: Should contributing.md text be moved to the repo readme.md ? -- less button clicks to navigate. Or at least add a blurb about the high fidelity developer fund.

Especially the telegram link for people wanting to know more.

Comment (misslivirose) [reply contributing.md]: We should probably update the Readme to reflect this doc / additional information about the developer fund, but keep this as a separate file too.

**Pull request closed without merging**